mqtt/mqtt-broker/Cargo.toml

```diff
@@ -9,6 +9,7 @@ async-trait = "0.1"
 bincode = "1.3"
 bytes = "1.0"
 chrono = { version = "0.4", features = ["serde"] }
+criterion = { version = "0.3", optional = true }
```

Comment:
You can still keep it in dev-deps. That's more appropriate for a dep of benches.

Reply:
Then we will need to change the feature below right? since features are with dependencies and not dev-dependencies? Maybe I am wrong with this, but that is my understanding at least

Reply:
Ah, right.